webapp/components/youtube_video.jsx

```diff
@@ -120,6 +124,13 @@ export default class YoutubeVideo extends React.Component {
     }
 
     render() {
+        if (this.state.failed) {
+            return <div/>;
```

Comment:
We also have to reserve the space in the failed case as well. Otherwise failed to load videos will cause jumping.